lsst/ts/cbp/cli.py

```diff
@@ -8,7 +8,7 @@
 
 
 @argh.arg('-ll', '--log-level', choices=['info', 'debug'])
-def start(log_level="info"):
+def start(log_level="info",simulation_mode=0):
```

Comment:
As a heads up: dds salobj (version 4) removes this argument from `start`. So you have that simplification to look forward to when you update.